webapp/packages/plugin-data-import/src/DataImportBootstrap.ts

```diff
@@ -103,7 +103,7 @@ export class DataImportBootstrap extends Bootstrap {
         const presentation = context.get(DATA_CONTEXT_DV_PRESENTATION);
         const isContainer = model.source instanceof ContainerDataSource;
         return (
-          !model.isReadonly(resultIndex) &&
```

Comment:
Fix is not clear for me. Why do we need this? Previous logic seems fine

Reply:
You probably need to use hasRowIdentifier flag to check whether table has pk

Reply:
this isReadonly method uses hasRowIdentifier inside, which we should ignore. How do you see this c check here?

Reply:
We probably want to remove this check from isReadonly as in fact table is editable

Reply:
No, this is not true